src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -404,13 +405,28 @@ export default class ConvasController {
     }
 
     /**
+     * Fill entire scope with the background color specified in the current theme
+     *
+     * @for CanvasController
+     * @method canvas_fill_background
+     * @param cc {Object} HTML5 canvas context
+     */
+    canvas_fill_background(cc) {
+        const SCOPE_THEME = this.theme.SCOPE;
+
+        cc.fillStyle = SCOPE_THEME.BACKGROUND;
```

Comment:
`SCOPE_THEME` is unnecessary here. You can simply use `this.theme.SCOPE.BACKGROUND`